### ISSUE TITLE
Add HTTPS

### ISSUE THREAD

**Issue by newswim** [CommonGarden/CommonGarden-Website#12]
Either by hosting the files on [Cloudflare] OR using something like [Netlify] with free integration with [Let's Encrypt].

[Cloudflare]: https://www.cloudflare.com/
[Netlify]: https://www.netlify.com
[Let's Encrypt]: https://letsencrypt.org/

**Comment by JakeHartnell** [comment]
Yeah, we're going to redo our website as it's very out of date, it will be HTTPS for sure. Will leave this issue open until it's done.
